frontend/src/pages/register/formValidation.tsx

```diff
@@ -86,41 +82,58 @@ const FormValidation = () => {
       });
   };
 
+  const handleAlertClose = () => {
+    setError('');
+    setSuccess('');
+  };
+
   return (
-    <Form.Base onSubmit={handleSubmit(submitForm)}>
-      <Holder m="0 auto" maxWidth="200px" minHeight="15px">
-        {error && <Error>{error}</Error>}
-        {success && <Success>{success}</Success>}
-      </Holder>
-      <InputFieldWrapper>
-        {registeFieldConfig.map((fieldConfig: RegisterFieldConfig) => (
-          <InputField
-            key={fieldConfig.name}
-            placeholder={fieldConfig.placeholder}
-            error={errors[fieldConfig.name]}
-            type={fieldConfig.type}
-            {...register(fieldConfig.name, fieldConfig.validations)}
-          />
-        ))}
-        <Holder flexDirection="column" height="45px">
-          <CheckBox
-            id="id"
-            {...register('areTermsRead', {
-              required: 'This field is required',
-            })}
-          >
-            <PlainText>
-              I accept{` `}
-              <Link to={Routes.TERMS_AND_CONDITIONS} target="_blank">
-                Terms of Service
-              </Link>
-            </PlainText>
-          </CheckBox>
+    <>
+      {error && (
+        <AlertContainer
+          type="error"
```

Comment:
error string, info string..:) see my point? You are going to avoid accidental typos if you create an enum